api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,189 @@
 			<column name="uuid" type="char(38)" />
 		</addColumn>
 	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1">
+		<preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="cohort_member" columnName="cohort_member_id"/></not>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="member_patient" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.1">
+		<preConditions onFail="MARK_RAN">
+			<not><columnExists tableName="cohort_member" columnName="cohort_member_id"/></not>
+		</preConditions>
+		<dropForeignKeyConstraint constraintName="parent_cohort" baseTableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-1.2">
+		<preConditions onFail="MARK_RAN">
+			<not><columnExists tableName="cohort_member" columnName="cohort_member_id"/></not>
+		</preConditions>
+		<comment>Dropping primary key for cohort_member</comment>
+		<dropPrimaryKey tableName="cohort_member"/>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="cohort_member_id"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="cohort_member_id" type="int">
+				<constraints nullable="false"/>
```

Comment:
I read addAutoIncrement adds the values incrementally for existing rows, since it literally translates to sql:

ALTER TABLE table.column MODIFY id INT AUTO_INCREMENT;

I tested this on a test table. So altering the table to add a primary key and auto increment in theory should fill the values? I can add the nullable constraint right after this.